packages/styleguide/src/templates/Article/base.js

```diff
@@ -62,7 +71,7 @@ const createBase = ({ metaBody, metaHeadlines }) => {
       }
       if (href && href[0] === '#') {
         return (
-          <Editorial.A
+          <LinkComponent
```

Comment:
see above – `ExpandableLink` does not have hash routing support with a custom on click – should be skipped.

```suggestion
          <Editorial.A
```